### PR TITLE
[BN2][Testing] Run latest version of Benchnet2 automation code

### PR DESCRIPTION
**Pull request by gomisha**
This PR automates the process of running the latest BN2 automation code while generating bootstrap info on a previous version of `flow-go` that will be deployed to BN2.

- clones an existing `flow-go` tagged version into a subdirectory of `integration/benchnet2`
- generates bootstrap info on the cloned `flow-go` tagged version but saves the bootstrap info outside of the cloned repo (in `integration/benchnet2`) so that the latest BN2 code can be run without requiring any copying of directories to the cloned repo

ref: https://github.com/dapperlabs/dapper-flow-hosting/issues/957

### CODE REVIEW

**Review by sjonpaulbrown — APPROVED** [review]

File: integration/benchnet2/Makefile [hunk 1]
```diff
@@ -49,11 +52,15 @@ gen-helm-values: validate gen-bootstrap gen-helm-l1 gen-helm-l2
...
 # main target for cleaning up a deployment
-clean-all: validate k8s-delete k8s-delete-secrets clean-bootstrap
+clean-all: validate k8s-delete k8s-delete-secrets clean-bootstrap clean-gen-helm clean-flow
 
 clean-bootstrap:
 	rm -rf ./bootstrap
 
+clean-gen-helm:
+	rm values.yml
+	rm template-data.json
```
Comment (SaveTheRbtz) [inline integration/benchnet2/Makefile:62]: maybe `rm -f`?

**Review by SaveTheRbtz — APPROVED** [review]

File: integration/benchnet2/Makefile (hunk 1 above)
Comment (gomisha) [reply integration/benchnet2/Makefile]: 9cf9a9d8bf6ed6e93e0a3d4feb66ba6d4100b42e

**Comment by gomisha** [comment]
bors merge

**Comment by gomisha** [comment]
bors merge

**Pull request merged**